src/index.js

```diff
@@ -26,45 +29,48 @@ server.on('connection', (socket) => {
 	logger.info(`CONNECTED: ${socket.remoteAddress}:${socket.remotePort}`)
 
 	socket.on('data', async (data) => {
-		
+
 		socket.end()
 		socket.destroy()
 
-		logger.debug('Received data', data.toString())
-
-		const {ip, port, username} = parser(data.toString())
-		logger.debug(`Parsed ${username} ${ip} ${port}`)
+		try {
```

Comment:
It may be better for readability to catch specific blocks who fails and describe why. Only catch the error won't give more information I think, but now it is handled with the logger which is nice.

What errors did you encounter ? It is possible that some parsing error has occurred, I had some while testing the app and of course transmitting text that is not formatted as described in parser.spec.js examples will fail and will throw an error.

Have you encountered other errors ? Such as writing to influx ?

Reply:
the error I encountered was ipLocation == null. You have already handled this with an explicit test, but I just wanted to catch everything.